api/src/pcapi/routes/serialization/offers_serialize.py

```diff
@@ -357,7 +357,8 @@ class Config:
 
 class GetStocksResponseModel(BaseModel):
     stocks: list[GetOfferStockResponseModel]
-    stock_count: int
+    stockCount: int
+    hasStocks: bool
```

Comment:
<!-- Issues represent user-facing problems. If possible, it's great to follow this kind of comment with a suggestion.  -->
**🔨 issue:** ‏Tu n'as pas besoin de camelCase ici
C'est même mieux de laisser en snake_case.

Pour que lke front reçoive un json avec du camelCase, tu ajoutes `alias_generator = serialization_utils.to_camel` à la class Config